From b637ef90120a9f94972050d35a7e15c59072b37c Mon Sep 17 00:00:00 2001
From: Sam Mortimer <sam@mortimer.me.uk>
Date: Tue, 28 Aug 2018 11:42:38 -0700
Subject: [PATCH 2/8] LineageSettingsProvider: Don't set
 LINEAGE_SETUP_WIZARD_COMPLETED

*) This was initialized from Settings.Global.DEVICE_PROVISIONED and it is no
   longer possible to access system settings during provider startup since:
https://android.googlesource.com/platform/frameworks/base/+/235272a2165b694ca1d67e901aa2ef2ad86c35bc

*) LINEAGE_SETUP_WIZARD_COMPLETED has been deprecated since 14.1:
https://github.com/LineageOS/android_lineage-sdk/commit/797a45aa6948e9c0871e47160fd6768bababee80

*) We will later cleanup the unused setting during topic:pie-db-upgrade

Change-Id: I185648ca25995fae80362642d297573795d04bfa
---
 .../org/lineageos/lineagesettings/LineageDatabaseHelper.java  | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/packages/LineageSettingsProvider/src/org/lineageos/lineagesettings/LineageDatabaseHelper.java b/packages/LineageSettingsProvider/src/org/lineageos/lineagesettings/LineageDatabaseHelper.java
index f245dd9..0581219 100644
--- a/packages/LineageSettingsProvider/src/org/lineageos/lineagesettings/LineageDatabaseHelper.java
+++ b/packages/LineageSettingsProvider/src/org/lineageos/lineagesettings/LineageDatabaseHelper.java
@@ -419,10 +419,6 @@ public class LineageDatabaseHelper extends SQLiteOpenHelper{
             loadStringSetting(stmt,
                     LineageSettings.Secure.PROTECTED_COMPONENT_MANAGERS,
                     R.string.def_protected_component_managers);
-
-            final String provisionedFlag = Settings.Global.getString(mContext.getContentResolver(),
-                    Settings.Global.DEVICE_PROVISIONED);
-            loadSetting(stmt, LineageSettings.Secure.LINEAGE_SETUP_WIZARD_COMPLETED, provisionedFlag);
         } finally {
             if (stmt != null) stmt.close();
         }
-- 
2.17.1

